### ISSUE TITLE
Research: Find out what SOAP is

### ISSUE THREAD

**Issue by CoolCat467** [CoolCat467/Scanner-Server#89]
On my home server running this program, I've noticed that windows devices seem to be trying to ask the scanner server about something. (soap-weirdness)[https://github.com/CoolCat467/Scanner-Server/tree/soap-weirdness] branch was created a while ago with a test for decoding a SOAP packet or whatever it is.

Specs pages I know about so far:
- https://specs.xmlsoap.org/ws/2005/04/discovery/ws-discovery.pdf
- https://specs.xmlsoap.org/ws/2004/09/policy/ws-policy0904.pdf
- https://specs.xmlsoap.org/ws/2005/07/securitypolicy/ws-securitypolicy.pdf

Basically we want to figure out how to respond to these SOAP packets in whatever way is best fit for this application.
